### DISCUSSION TITLE
Installation problems Vue

### DISCUSSION THREAD

**Discussion by frontero** [nolimits4web/swiper#5563] [Q&A]
Hi! I've done everything on: https://swiperjs.com/vue#installation

There is a folder 'swiper' in my node_modules containing a folder 'vue'.

` ERROR  Failed to compile with 1 error                                                                                  21:10:53

This dependency was not found:

* swiper/vue in ./node_modules/cache-loader/dist/cjs.js??ref--13-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/d
ist/cjs.js??ref--1-0!./node_modules/vue-loader/lib??vue-loader-options!./src/components/sections/swiper-cards.vue?vue&type=scrip
t&lang=js&

To install it, you can run: npm install --save swiper/vue
`

When I try to run `npm install --save swiper/vue` I get a popup from Github. After I've entered my credentials on the browser it says connected and then that's it. Nothing in the terminal. How can I get this up and running? I have the node_modules/swiper/vue folder.

**Comment by Arhell** [comment]
Try to use the command which is in the documentation
`npm i swiper`

Also remove node_modules and package.json lock
and reinstall

https://stackoverflow.com/questions/69179230/unable-to-use-swiper-vue-dependencies-not-found
